src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -108,6 +112,15 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	get activeCellId(): string {
 		return this._activeCellId;
 	}
+	/**
+	 * Returns the code editor of makrdown cell in edit mode.
+	 */
+	getEditor(): BaseTextEditor | undefined {
```

Comment:
Does this impact find decorations?

Reply:
No, it doesn't since I find the editor from cell guid and the code editor is a different cellModel itself.

Reply:
I have tested on my tests branch, it does impact find since the cell guid is same for the code editor and the html component. I made some changes for find to work as usual. Thanks Chris for catching it!